x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/connector/syncjob/action/UpdateConnectorSyncJobIngestionStatsActionRequestTests.java

```diff
@@ -39,7 +39,8 @@ public void testValidate_WhenConnectorSyncJobIdIsEmpty_ExpectValidationError() {
             0L,
             0L,
             0L,
-            Instant.now()
+            Instant.now(),
+            null
```

Comment:
Would it be useful to have a test that includes some example metadata among the tests in this file?

Reply:
In the file [ConnectorSyncJobTestUtils.java](https://github.com/elastic/elasticsearch/pull/109927/files#diff-54c2ba67874d1ca82cea5afb3de24a79cc544353cec508419afd7e97e9516f60) you can see metadata as a `randomMap` in `getRandomUpdateConnectorSyncJobIngestionStatsActionRequest` function. But can add a few tests in here as well 👍